keps/sig-auth/541-external-credential-providers/README.md

```diff
@@ -710,16 +825,19 @@ _This section must be completed when targeting beta graduation to a release._
 
 * **What are the SLIs (Service Level Indicators) an operator can use to
   determine the health of the service?**
-  - [ ] Metrics
-    - Metric name:
-    - [Optional] Aggregation method:
-    - Components exposing the metric:
-  - [x] Other (treat as last resort)
+  - [X] Metrics
+    - Metric name: `rest_client_exec_plugin_ttl_seconds`, `rest_client_exec_plugin_certificate_rotation_age`,
+      `rest_client_exec_plugin_calls`, `rest_client_exec_plugin_failed_calls`
+    - Components exposing the metric: client-go
+  - [ ] Other (treat as last resort)
     - Details:
       - This feature set operates on the client-side.
 
 * **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
-  - This feature set operates on the client-side.
+  - `rest_client_exec_plugin_ttl_seconds`: the expected lifetime of client-side certificates, in seconds
+  - `rest_client_exec_plugin_certificate_rotation_age`: the expected lifetime of client-side certificates, in seconds
+  - `rest_client_exec_plugin_calls`: 1 per the lifetime of the credential returned by the exec plugin
+  - `rest_client_exec_plugin_failed_calls`: 0, or a very low number compared to `rest_client_exec_plugin_calls`
```

Comment:
Ah, yes, I do see that I flipped that around. I made another pass at this section (and verbatim used your example, since it seems like a reasonable failure rate to me). I am trying to look at this like you said: start with the operations outcome, and then answer how to achieve that outcome with specific metrics. I still feel a little shaky on my first SLO, but I figured it would be better for me to get something down on paper for us to discuss sooner rather that later.

https://github.com/kubernetes/enhancements/pull/2275/files